package.json

```diff
@@ -127,5 +127,12 @@
         "file": "HEADER"
       }
     }
+  },
+  "dependencies": {
+    "@accordproject/concerto-cto": "3.19.2",
+    "axios": "1.7.7",
+    "jsome": "2.3.26",
+    "pdfjs-dist": "4.7.76",
+    "webpack": "5.95.0"
```

Comment:
Why have these dependencies been added?

Reply:
I ran `npm audit fix`

Reply:
We shouldn't need to add any new deps, only update some of them. There should be no downgrades either.